$OpenBSD: patch-wscript,v 1.3 2015/07/16 01:26:16 jca Exp $

- samba is unusable (slow startup) without --as-needed; no visible drawback yet

--- wscript.orig	Wed Jul 15 17:52:28 2015
+++ wscript	Wed Jul 15 18:27:29 2015
@@ -154,9 +154,7 @@ def configure(conf):
     # strict as the strictest OS we support, so adding this here
     # allows us to find problems on our development hosts faster.
     # It also results in faster load time.
-
-    if not sys.platform.startswith("openbsd"):
-        conf.env.asneeded_ldflags = conf.ADD_LDFLAGS('-Wl,--as-needed', testflags=True)
+    conf.env.asneeded_ldflags = conf.ADD_LDFLAGS('-Wl,--as-needed', testflags=True)
 
     if not conf.CHECK_NEED_LC("-lc not needed"):
         conf.ADD_LDFLAGS('-lc', testflags=False)
